api/app/Modules/Core/Transformers/Transformer.php

```diff
@@ -41,10 +41,10 @@ protected function timestamps(TimestampedEntity $entity): MergeValue
     }
 
     /**
-     * @return \League\Fractal\Resource\NullResource|Primitive
+     * @return NullResource
      */
-    protected function null()
+    protected function null(): NullResource
     {
-        return $this->primitive(null);
+        return new NullResource();
```

Comment:
This is causing issues in the frontend. For tracks with no lyrics, the response has changed from:

```
"lyrics": null,
```

to 
```
"lyrics": [],
```

That's why this URL is breaking, along with many others: https://pr-592.intg.k8s.nawhas.com/reciters/farhan-ali-waris/albums/2008/tracks/ab-tera-bichona

I've pushed a commit (234f8563) to fix this, but just an FYI